src/style.css

```diff
@@ -0,0 +1,163 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital@1&family=Roboto+Slab:wght@900&family=Roboto:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital@1&family=Roboto+Slab:wght@900&family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  border: 0;
+  margin: 0;
+  padding: 0;
```

Comment:
better to reset it for the list of tags you need instead of put it to `*`

Reply:
fix this comment, and add reset directly to that tag that you need
https://github.com/mate-academy/layout_product-cards/pull/2400/files#r1066038842

Reply:
last comment is more clear, thanks , delete it all out  ,as no influence , in general